editor/components/page-attributes/parent.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import { get } from 'lodash';
+import { stringify } from 'querystringify';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { withInstanceId, withAPIData } from '@wordpress/components';
+import { compose } from '@wordpress/element';
+import { buildTermsTree } from '@wordpress/utils';
+
+/**
+ * Internal dependencies
+ */
+import { getCurrentPostId, getEditedPostAttribute, getCurrentPostType } from '../../store/selectors';
+import { editPost } from '../../store/actions';
+import TermTreeSelect from '../../../blocks/term-tree-select/index.js';
```

Comment:
Oh I guess this is the "import" breaking the styling, since we're importing from another module here, we're probably changing the style loading order.

I guess this should be something like `import { TermTreeSelect } from '@wordpress/blocks'`? or better, maybe we should move this component to `@wordpress/components` and rename it just `TreeSelect`?

Reply:
That opens a bigger discussion,  TermTreeSelect uses '../inspector-controls/select-control'; which uses BaseControl from blocks/inspector-controls/base-control/. I think we should move all of inspector-controls "input fields" to @wordpress/components. This input fields would also be useful for the editor in the document sidebar e.g they handle label "for" and id generation for us. On inspector controls, we make use of that, on document sidebar we are repeating this logic for each input field.

Reply:
That makes sense to me but let's do it a separate PR and discuss with everyone. Also, if we do so, we need to provide a way to use them from `wp.blocks.*` for some time and warn about deprecation.